examples/example-gateway/endpoints/tchannel/baz/baz_call.go

```diff
@@ -68,7 +68,7 @@ func (w Workflow) Handle(
 	}
 
 	clientRespHeaders, err := w.Clients.Baz.Call(ctx, clientReqHeaders, clientReq)
-	respHeaders := zanzibar.ServerTChannelHeader(clientRespHeaders)
```

Comment:
It seems there is  a chance that when casting to zanzibar.ServerTChannelHeader, it might return a nil, which would be triggers NPE at [here](https://github.com/uber/zanzibar/blob/5274bc56f715972efb39b3665a5104d6e2a34fe6/examples/example-gateway/build/endpoints/tchannel/baz/baz_simpleservice_method_call_tchannel.go#L91)

Reply:
If the link is right, `wfResHeaders, err := workflow.Handle(ctx, wfReqHeaders, &req)` doesn't seem like where a NPE would happen.

Reply:
Sorry here is the correct https://github.com/uber/zanzibar/blob/5274bc56f715972efb39b3665a5104d6e2a34fe6/examples/example-gateway/build/endpoints/tchannel/baz/baz_simpleservice_method_call_tchannel.go#L94

Reply:
Why would that line cause NPE? Calling `Keys()` on a nil header is okay, see https://play.golang.org/p/HsRL1EzYZ0e